auth/src/main/java/com/firebase/ui/auth/util/ui/BaselineTextInputLayout.java

```diff
@@ -1,11 +1,13 @@
 package com.firebase.ui.auth.util.ui;
 
 import android.content.Context;
-import android.support.annotation.RestrictTo;
-import android.support.design.widget.TextInputLayout;
 import android.util.AttributeSet;
 import android.widget.EditText;
 
+import com.google.android.material.textfield.TextInputLayout;
+
+import androidx.annotation.RestrictTo;
+
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class BaselineTextInputLayout extends TextInputLayout {
```

Comment:
This can finally die! 🎉 #1464

Reply:
For sure!  Gonna do that in a separate PR, I want this one to be as "mechanical" as possible.